src/components/scene/device-motion-permission-ui.js

```diff
@@ -0,0 +1,121 @@
+var registerComponent = require('../../core/component').registerComponent;
+var utils = require('../../utils/');
+var bind = utils.bind;
+
+var constants = require('../../constants/');
+
+var DEVICE_PERMISSION_CLASS = 'a-device-motion-permission';
+var DEVICE_PERMISSION_BTN_CLASS = 'a-device-motion-permission-button';
+
+/**
+ * UI for enabling device motion permission
+ */
+ module.exports.Component = registerComponent('device-motion-permission', {
+
+  dependencies: ['canvas'],
+
+  schema: {
+    enabled: {default: true},
+    enableFunc: {default:''}
```

Comment:
What is this for?

Reply:
These are in flux and are not yet implemented. They may not belong in the final PR on this. I will revisit before resubmitting.